public/index.html

```diff
@@ -652,6 +664,19 @@ <h2 class="mb-5">Locale</h2>
             event.preventDefault();
         }
 
+        function executeFunction() {
+            let promise = functions.createExecution('test', '', true);
+
+            promise.then(
+                function (response) {
+                    alert("Function executed successfully!");
+                },
+                function (error) {
+                    alert("Failed to execute function.");
+                }
+            );
+        }
+
```

Comment:
@vermakhushboo there should also be code to create the deployment and activate it

Reply:
I don’t think web SDK allows user to create function or deployment. It only has create execution and list executions methods. So we need to create that function and deployment manually to test this flow.

Reply:
Added comment explaining how to create function